### PR TITLE
Website - Kelsey McAlpine

### PR DESCRIPTION
**Pull request by ghost**
# Static Site
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| Did you have to resolve any issues when running the HTML Validator? If so, what were they?  |  I'm not sure if I did it right?  |
| Why is it important to consider and use semantic HTML? |  Because it helps give meaning to the structure of the page.   |
| How did you decide to structure your CSS? | I started with global elements that are able to be applied to every page, and then for each page I created a section under a comment that includes css for that specific page.  |
| What was the most challenging piece of this assignment? |  Figuring out which display type to use, and using floats.  |
| Describe one area that you gained more clarity on when completing this assignment |  How to utilize the developer tools. |

### CODE REVIEW

**Comment by kariabancroft** [comment]
# Static Site
## What We're Looking For

| Feature | Feedback    |
| :------------- | :------------- |
|  **Baseline** |   |
|   Appropriate Git Usage	|   Yes	|
|   Answered comprehension questions	| Yes. You mention one important reason to use semantic HTML but one that you didn't mention is _accessibility_.	|
|   Page fully loads |Yes |
|   No broken links (regular or images) | Yes (there is one broken link but I think it's a lorem issue, not yours) |
|   Includes at least 4 pages and styling |Yes |
|  **HTML** |   |
|   Uses the high-level tags for organization: `header, footer, main` | Yes |
|   Appropriately using semantic tags: `section, article`, etc. | Yes|
|   All images include alternate text |Yes |
|  **CSS** |   |
|   Using class and ID names in style declarations |Yes. I like the way you used classes for post and content that are similar. Also using IDs for some specific scenarios. |
|   Style declarations are DRY  | Yes - nothing really stood out to me as opportunity to DRY up duplicate styles across the board. Nice job.|
|  **Overall** |  I like the way you've sectioned each part of your CSS so it's easy to follow what each set of selectors is using. Nice job focusing on the learning of CSS rather than the content of your website. Overall very pleasing design.  |

File: about.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <li><h3><a href="blog.html">Blog</a></h3></li>
+          <li><h3><a href="about.html">About</a></h3></li>
+          <li><h3><a href="index.html">Work</a></h3></li>
+        </ul>
+      </section>
+    </header>
+    <main>
+      <section class="container">
+        <img src="http://lorempixel.com/g/400/500" alt="placeholder">
+        <section id="about-info">
```
Comment (goaskcarmen) [inline about.html:32]: This section changes its position with different screen resolutions. If you make the screen smaller, it drops to the bottom of the image. No big deal. You will learn how to use Foundation and/or Bootstrap to make your screen more responsive later.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <section id="header-top">
+        <h5 class="left">FULL-STACK DEVELOPER | SEATTLE, WA</h5>
+        <ul class="right">
+          <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+          <li><a href="#"><i class="fa fa-github"></i></a></li>
+        </ul>
+      </section>
+      <section id="header-bottom">
+        <h1 class="left">KELSEY MCALPINE</h1>
```
Comment (goaskcarmen) [inline about.html]: This is a personal preference. I would make this a link to bring the user back to the Homepage (index).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    <section id="header-bottom">
+      <h1 class="left">KELSEY MCALPINE</h1>
+      <ul class="right">
+        <li><h3><a href="blog.html">Blog</a></h3></li>
+        <li><h3><a href="about.html">About</a></h3></li>
+        <li><h3><a href="index.html">Work</a></h3></li>
+      </ul>
+    </section>
+  </header>
+  <main>
```
Comment (goaskcarmen) [inline index.html:29]: Good job on centering the images and switching the images each time the page is refreshed.

**Review by goaskcarmen — COMMENTED** [review]
Your website looks great! I really like how you section your Blog. Sorry, I don't have a lot of comments on this project. I am more of a back-end person. If you have any question on Rails, feel free to  email me.

**Pull request closed without merging**